gnomad_qc/v4/analyses/grpmax_comps.py

```diff
@@ -0,0 +1,419 @@
+        else:
+            ht = get_gnomad_public_data("exomes")
+
+        if args.test:
+            # Filter to two partitions
+            ht = ht._filter_partitions([0, 1])
+        t_variants = ht.count()
+
+        logger.info(
+            "Total number of variants in %s before filtering: %s", version, t_variants
+        )
+        msg = ""
+        ht = process_consequences(ht, has_polyphen=False)
+
+        if csq_terms:
+            logger.info(
+                "Filtering to keep only %s ...",
+                (
+                    str(csq_terms) + " variants"
+                    if csq_terms and not non_syn_only
+                    else "non-synonymous variants"
+                ),
+            )
+        if args.canonical:
+            vep_csq_expr = ht.vep.worst_csq_for_variant_canonical
+            msg += " on canonical transcripts"
+        else:
+            vep_csq_expr = ht.vep.worst_csq_for_variant
+            msg += " on all transcripts"
+
+        ht = ht.filter(
+            hl.literal(csq_terms).contains(vep_csq_expr.most_severe_consequence)
+        )
+        ns_variants = ht.count()
+
+        logger.info(
+            "Total number of variants in %s %s: %s (%.2f%% of total variants)",
+            version,
+            "after " + msg if msg else "",
+            ns_variants,
+            (ns_variants / t_variants) * 100,
+        )
+
+        ht = ht.checkpoint(
+            f"gs://gnomad-tmp-4day/grp_comps_{version}_canonical_non_syn.ht",
+            overwrite=True,
+        )
+
+        if args.no_variant_qc_filters:
+            msg = "of variants, regardless of variant QC status"
+        elif args.drop_hard_filtered:
+            # Filter out AC0 and InbreedingCoeff variants
+            ht = ht.filter(~ht.filters.any(lambda x: FILTER_VALUES_TO_DROP.contains(x)))
+            msg = "of variants passing ONLY hard filters (RF/VQSR are retained)"
+        else:
+            # Filter to only PASS
+            ht = ht.filter(hl.len(ht.filters) == 0)
+            msg = "of variants passing all filters"
+
+        p_ns_variants = ht.count()
+        logger.info(
+            "Total number %s: %s (%.2f%% of total variants)",
+            msg,
+            p_ns_variants,
+            (p_ns_variants / t_variants) * 100,
+        )
+
+        version_dict[version] = version_stats(
+            ht,
+            t_variants=t_variants,
+            version=version,
+            grpmax_counts=grpmax_counts,
+            eur_filter=eur_filter,
+        )
+
+    if grpmax_counts and eur_filter:
+        for eur_grp in EUR_GRPS.keys():
+            create_table(
+                version_dict,
+                data_subset=eur_grp,
+                grpmax_counts=grpmax_counts,
+                non_syn_only=non_syn_only,
+                eur_filter=eur_filter,
+            )
+    else:
+        create_table(
+            version_dict, grpmax_counts=grpmax_counts, non_syn_only=non_syn_only
```

Comment:
```suggestion
            version_dict, non_syn_only=non_syn_only
```
also very minor -- just seems a little clearer to not specify either `grpmax_counts` or `eur_filter` based on the if/else logic

Reply:
So I kept this in so you can run it without the eur filter and still get overall grpmax counts if you want, or if you dont care to deduplicate, you can still get the AF>threshold counts